sci-mathematics/gmp_ecm/gmp_ecm-7.0.4.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Elliptic Curve Method for Integer Factorization"
+DESCRIPTION="C implementation of the Elliptic Curve Method for factorizing integers"
+HOMEPAGE="http://ecm.gforge.inria.fr/"
+COPYRIGHT="2001-2016 GMP-ECM Authors"
+LICENSE="
+	GNU GPL v3
+	GNU LGPL v3
+	"
+REVISION="1"
+SOURCE_URI="https://gforge.inria.fr/frs/download.php/file/36224/ecm-$portVersion.tar.gz"
+CHECKSUM_SHA256="0cf7b3eee8462cc6f98b418b47630e1eb6b3f4f8c3fc1fb005b08e2a1811ba43"
+SOURCE_DIR="ecm-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gmp_ecm$secondaryArchSuffix = $portVersion
+	cmd:ecm$secondaryArchSuffix = $portVersion
+	lib:libecm$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gmp_ecm${secondaryArchSuffix}_devel = $portVersion
+	devel:libecm$secondaryArchSuffix = $portVersion
```

Comment:
I think it should be
```
	devel:libecm$secondaryArchSuffix = $libVersionCompat
```
or maybe
```
	devel:libecm$secondaryArchSuffix = $libVersion
```
I don't know which is correct. @korli (or any other expert), could you help us here? Thanks!

Reply:
I would prefer `$libVersion`. While minor bumps may not break ABI compatibility, they may introduce new features, and we would like to use them

Reply:
Sorry for this late reply. I was doing some stats about how many existing recipes have compat for their `devel:lib*`lines in PROVIDES_devel and how many don't have, but I could not figure out if there is a best practice about this.
Anyway, since we have many with, and many without, please feel free to do as you think is best.

Hot (good) news! If my find+grep+sed based scripts are not too wrong, it seems we have twice more recipes *without* compat versions (in PROVIDES_devel) than we have *with* compat versions (in PROVIDES_devel).